phpBB/memberlist.php

```diff
@@ -1147,6 +1147,20 @@
 					$user_rank_data['img'] .= '<br />';
 				}
 			}
+			// include modules for manage groups link display or not
+			// need to ensure the module is active
```

Comment:
It is, pretty much, no different than viewing a profile (p_master is called then too).  As far as overhead without the edits I get `Queries: 13 | Peak Memory Usage: 10.02 MiB` with the edits I get on first run `Queries: 14 | Peak Memory Usage: 10.21 MiB` one additional query to build the cache and .19mb of memory.  Could add a `if user_logged in` check to only run for registered users but the "overhead" seems a bit minimal to me.  It would seem that visiting a user profile creates more overhead than this does?

Up to devs really, can't really see another way to accomplish the task and handle the ticket request.